src/Qowaiv/Clock.cs

```diff
@@ -38,14 +38,14 @@ public static class Clock
     [Pure]
     public static DateTime UtcNow()
     {
-        var utcNow = (threadUtcNow.Value ?? globalUtcNow).Invoke();
+        var utcNow = (localUtcNow.Value ?? globalUtcNow).Invoke();
```

Comment:
'local' probably means something like local to the current "execution" context. But the term local is also used within the time domain wrt timezones and thus not very descriptive.
I'm struggling with clear terminologie as well. Some suggestions:
Alternative 1) Use the term `asyncLocalUtcNow`, to indicate it is local to the current execution context.
Alternative 2) localContextUtcNow,
Alternative 3) currentContextUtcNow

Reply:
@Sjaaky localContextUtcNow might be a better suggestion indeed.